extensions/ql-vscode/package.json

```diff
@@ -209,6 +209,24 @@
           "default": null,
           "description": "Path to a directory where the CodeQL extension should store query server logs. If empty, the extension stores logs in a temporary workspace folder and deletes the contents after each run."
         },
+        "codeQL.runningQueries.structuredEvaluatorLog": {
+          "type": [
+            "string",
+            null
+          ],
+          "default": "structured-evaluator-log.json",
+          "description": "Path to a file where the evaluator should store structured evaluator logs. If empty, the query server stores the logs in a default location and overwrites them after each query server run."
+        },
+        "codeQL.runningQueries.structuredEvaluatorLog.minify": {
+          "type": "boolean",
+          "default": false,
+          "description": "Minifies the JSON output of structured evaluator logs, removing all whitespace except a single newline between event objects. If not passed, logs are not minified."
+        },
+        "codeQL.runningQueries.structuredEvaluatorLog.verbosity": {
+          "type": "integer",
+          "default": 1,
```

Comment:
I think we might just want to make the extension always pass in the highest verbosity. As we develop features of the extension to display different information from the structured logs, it will be annoying if the information may or may not be present depending on some user-supplied value. It seems easier to just make sure all the information is always there.